Update consider_using_a_fluent_api.en.md

kathyrollo opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->
Remove implementation for `LoadableComponent<>` in `GoogleSearchPage` example.

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
The fluent interface can be implemented much simpler while still conforming to the page/component object principles. `FindBy` should be discouraged and lambdas as `WebDriverWait` conditions should be encouraged as the preferred/modern practice over `ExpectedConditions` in Selenium 4.x.

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

kathyrollo committed:
Update consider_using_a_fluent_api.en.md

The Fluent Interface can be implemented much simpler to the same effect.
FindBy should no longer be encouraged in examples with Selenium 4.x.
Lambdas as wait conditions can be encouraged in examples as the more modern approach with Selenium 4.x.

kathyrollo committed:
Fix constructor in example

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1297) <br/>All committers have signed the CLA.

diemol reviewed (commented):
Thank you, @kathyrollo, I think this makes sense.

Could you please update the code in the related pages?  `consider_using_a_fluent_api.*.md`

kathyrollo commented:
> Thank you, @kathyrollo, I think this makes sense.
> 
> Could you please update the code in the related pages? `consider_using_a_fluent_api.*.md`

Thank you @diemol . Will take a look at the related pages in the following days.

kathyrollo committed:
Remove Assertion from page object

Selenium encourages to "generally not throw assertions" within pages objects or page components. The WebDriverWait throws an exception to the same effect.

kathyrollo committed:
Update code example in Fluent API (EN)

kathyrollo committed:
Update code example in Fluent API (JA)

kathyrollo committed:
Update code example in Fluent API (PT-BR)

kathyrollo committed:
Update code example in Fluent API (ZH-CN)

kathyrollo commented:
Dear @diemol , the related pages have been updated as requested. Cheers!

diemol reviewed (approved):
Thank you, @kathyrollo!
